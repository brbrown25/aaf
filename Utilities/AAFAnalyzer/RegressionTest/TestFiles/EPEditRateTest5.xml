<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE aaf-file SYSTEM "EPDerivationChain.dtd">

<aaf-file>

    <!-- Test all static tracks -->

    <clip name="Clip 1">
        <source-clip slot-type="static" edit-rate-denominator="1" edit-rate-numerator="1" track-type="audio">
            <file-source name="File Source 1" sample-rate-denominator="24" sample-rate-numerator="600">
                <eoc/>
            </file-source>
        </source-clip>
    </clip>
    <clip name="Clip 2">
        <source-clip slot-type="static" edit-rate-denominator="1" edit-rate-numerator="1" track-type="audio">
            <file-source name="File Source 1" sample-rate-denominator="24" sample-rate-numerator="1200000">
                <eoc/>
            </file-source>
        </source-clip>
    </clip>

</aaf-file>
