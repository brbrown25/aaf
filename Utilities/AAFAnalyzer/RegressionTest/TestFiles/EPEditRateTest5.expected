        Name:   Edit Protocol Edit Rate Test
        Desc:   Verify that all essence tracks have valid edit rates.
        Result: Failed
        Reason: Test Failed - See "Edit Protocol Edit Rate Visitor" Visitor for details
        Passing Requirements: REQ_EP_092; REQ_EP_099; REQ_EP_100
        Warning Requirements: None
        Failing Requirements: REQ_EP_091

            Name:   Edit Protocol Edit Rate Visitor
            Desc:   Ensure all essence tracks have valid edit rates.
            Result: Failed
            Reason: 
            Passing Requirements: REQ_EP_092; REQ_EP_099; REQ_EP_100
            Warning Requirements: None
            Failing Requirements: REQ_EP_091

                Name:   Edit Protocol Edit Rate Visitor
                Desc:   -
                Result: Failed
                Reason: Mob Slot with ID = 1 of Mob: "Clip 1" is a StaticMobSlot and does not have an edit rate.
                Passing Requirements: None
                Warning Requirements: None
                Failing Requirements: REQ_EP_091

                Name:   Edit Protocol Edit Rate Visitor
                Desc:   -
                Result: Failed
                Reason: Mob Slot with ID = 1 of Mob: "Clip 2" is a StaticMobSlot and does not have an edit rate.
                Passing Requirements: None
                Warning Requirements: None
                Failing Requirements: REQ_EP_091

